packages/schema/exported-schema.json

```diff
@@ -51,6 +51,10 @@
           "description": "All the triggers for your app. You can add your own here, or Zapier will automatically register any from the list/hook methods on your resources.",
           "$ref": "/TriggersSchema"
         },
+        "reads": {
```

Comment:
I'd probably call this `bulkReads` to match the name better

Reply:
Sounds good!